### DISCUSSION TITLE
Qiitadonのドメインの今後について

### DISCUSSION THREAD

**Discussion by yumetodo** [increments/qiita-discussions#130] [要望・アイデア・不具合]
[Qiitadonのサービス提供を終了します - Qiita Blog](https://blog.qiita.com/terminate-provision-of-qiitadon/)

ながらくサービス提供いただきありがとうございました。

Qiitadonのサービス終了後、 `qiitadon.com`のドメインがどうなるのか気になっています。すぐに手放すとなんらかの悪用が起きたりしないか気になります。

**Comment by kskwtnk** [comment]
@yumetodo 

ご意見ありがとうございます！

サービス終了後すぐに手放す予定はありませんが、ある程度の期間が経った後に手放す可能性はあります
ただし、いつまでは弊社が持つか……といったスケジュールは未定かつ、外部に告知はしない予定でいます
ご了承いただけますと幸いです :pray:

**Reply by yumetodo** [reply]
回答ありがとうございました！終了まで残り僅かですがQiitadonをよろしくおねがいします。
